Xamarin.Forms.Platform.iOS/Material/MaterialFrameRenderer.cs

```diff
@@ -1,51 +1,65 @@
 using System;
 using System.ComponentModel;
-using System.Drawing;
-using CoreAnimation;
-using MaterialComponents;
 using UIKit;
 using Xamarin.Forms;
 using MCard = MaterialComponents.Card;
 
+// this won't go here permanently it's just for testing at this point
 [assembly: ExportRenderer(typeof(Xamarin.Forms.Frame), typeof(Xamarin.Forms.Platform.iOS.Material.MaterialFrameRenderer), new[] { typeof(VisualRendererMarker.Material) })]
+
 namespace Xamarin.Forms.Platform.iOS.Material
 {
 	public class MaterialFrameRenderer : MCard, IVisualElementRenderer
```

Comment:
iOS needs some research to determine what the best way to crop the contents to fit the curve.